### ISSUE TITLE
Makefile: 194: Recipe execution error for target “php_memcached.lo”

### ISSUE THREAD

**Issue by denisgolius** [php-memcached-dev/php-memcached#437]
root@hosting:/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7# make
/bin/sh /usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/libtool --mode=compile cc -I/opt/php-7.3.5/include/php    -I. -I/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7 -DPHP_ATOM_INC -I/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/include -I/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/main -I/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7 -I/opt/php-7.3.5/include/php -I/opt/php-7.3.5/include/php/main -I/opt/php-7.3.5/include/php/TSRM -I/opt/php-7.3.5/include/php/Zend -I/opt/php-7.3.5/include/php/ext -I/opt/php-7.3.5/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2 -I/opt/php-7.3.5/include/php -I/opt/php-7.3.5/include/php/main -I/opt/php-7.3.5/include/php/TSRM -I/opt/php-7.3.5/include/php/Zend -I/opt/php-7.3.5/include/php/ext -I/opt/php-7.3.5/include/php/ext/date/lib   -c /usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/php_memcached.c -o php_memcached.lo 
mkdir .libs
 cc -I/opt/php-7.3.5/include/php -I. -I/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7 -DPHP_ATOM_INC -I/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/include -I/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/main -I/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7 -I/opt/php-7.3.5/include/php -I/opt/php-7.3.5/include/php/main -I/opt/php-7.3.5/include/php/TSRM -I/opt/php-7.3.5/include/php/Zend -I/opt/php-7.3.5/include/php/ext -I/opt/php-7.3.5/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -I/opt/php-7.3.5/include/php -I/opt/php-7.3.5/include/php/main -I/opt/php-7.3.5/include/php/TSRM -I/opt/php-7.3.5/include/php/Zend -I/opt/php-7.3.5/include/php/ext -I/opt/php-7.3.5/include/php/ext/date/lib -c /usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/php_memcached.c  -fPIC -DPIC -o .libs/php_memcached.o
/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/php_memcached.c: In function ‘php_memc_printable_func’:
/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/php_memcached.c:454:25: warning: format ‘%s’ expects argument of type ‘char *’, but argument 5 has type ‘zend_string * {aka struct _zend_string *}’ [-Wformat=]
   spprintf (&buffer, 0, "%s::%s", ZSTR_VAL(fci->object->ce->name), fci_cache->function_handler->common.function_name);
                         ^
/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/php_memcached.c: In function ‘zim_Memcached___construct’:
/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/php_memcached.c:1282:20: error: lvalue required as left operand of assignment
   GC_REFCOUNT(&le) = 1;
                    ^
/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/php_memcached.c: In function ‘s_stat_execute_cb’:
/usr/local/src/php7.3.5-build/php-memcache/php-memcached-php7/php_memcached.c:2640:24: warning: field precision specifier ‘.*’ expects argument of type ‘int’, but argument 4 has type ‘size_t {aka long unsigned int}’ [-Wformat=]
  spprintf (&buffer, 0, "%.*s", value_length, value);
                        ^
Makefile:194: ошибка выполнения рецепта для цели «php_memcached.lo»
make: *** [php_memcached.lo] Ошибка 1

**Comment by Arxcis** [comment]
Got same bug when installing `php-memcached` from [inside the `laradock/php-fpm`-Dockerfile](https://github.com/laradock/laradock/blob/master/php-fpm/Dockerfile#L355)

![image](https://user-images.githubusercontent.com/3092618/63230142-17205580-c209-11e9-8c17-04ea25330647.png)
```
 cc -I/usr/local/include/php -I. -I/var/www/html/memcached -DPHP_ATOM_INC -I/var/www/html/memcached/include -I/var/www/html/memcached/main -I/var/www/html/memcached -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib -c /var/www/html/memcached/php_memcached.c  -fPIC -DPIC -o .libs/php_memcached.o
/var/www/html/memcached/php_memcached_session.c: In function 'ps_open_memcached':
/var/www/html/memcached/php_memcached_session.c:378:20: error: lvalue required as left operand of assignment
   GC_REFCOUNT(&le) = 1;
                    ^
make: *** [php_memcached_session.lo] Error 1
make: *** Waiting for unfinished jobs....
Makefile:202: recipe for target 'php_memcached_session.lo' failed
/var/www/html/memcached/php_memcached.c: In function 'zim_Memcached___construct':
/var/www/html/memcached/php_memcached.c:1282:20: error: lvalue required as left operand of assignment
   GC_REFCOUNT(&le) = 1;
                    ^
Makefile:194: recipe for target 'php_memcached.lo' failed
make: *** [php_memcached.lo] Error 1
```

**Comment by calvaradocl** [comment]
Experiencing the same issue while trying to build:

```
/home/bitnami/php-memcached/php_memcached.c: In function ‘zim_Memcached___construct’:
/home/bitnami/php-memcached/php_memcached.c:1282:20: error: lvalue required as left operand of assignment
   GC_REFCOUNT(&le) = 1;
```

**Comment by sodabrew** [comment]
This was fixed in #390 about eleven months ago, so you have an out-of-date Dockerfile, or the Dockerfile has an out-of-date cached layer with an old `master` branch snapshot baked into it. Use php-memcached 3.1.3 because that's the most recent version and doesn't have this problem.

**Comment by Arxcis** [comment]
@sodabrew Thank you very much for submitting https://github.com/laradock/laradock/pull/2309 👍
